packages/debugger/lib/ast/sagas/index.js

```diff
@@ -7,31 +7,72 @@ import * as data from "lib/data/sagas";
 
 import ast from "../selectors";
 
-import flatten from "lodash.flatten";
 import jsonpointer from "json-pointer";
 
-function* walk(compilationId, sourceId, node, pointer = "", parentId = null) {
+function* walk(
+  compilationId,
+  internalFor,
```

Comment:
I think this identifier could be more clear - presumably it refers to a context?

Reply:
Well, it's gone now; `compilationId` and `internalFor` have been replaced by the lone `sourceId`. :)